kwanghyun/controllers/postControllers.js

```diff
@@ -0,0 +1,67 @@
+const postService = require("../services/postService");
+
+const createPosts = async (req, res) => {
+  try {
+    const { title, content, userId } = req.body;
+
+    if (!title || !content || !userId) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    await postService.createPosts(title, content, userId);
+
+    res.status(201).json({ message: "POSTING_SUCESS" });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const postslist = async (req, res) => {
+  try {
+    const rows = await postService.postslist();
+    return res.status(200).json({ rows });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const modify = async (req, res) => {
+  try {
+    const { content, postId } = req.body;
+
+    if (!postId || !content) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    await postService.modify(content, postId);
+    res.status(200).json({ message: "MODIFY_SUCESS" });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.messasge });
+  }
+};
+
+const postsDelete = async (req, res) => {
+  try {
+    const { postId } = req.body;
```

Comment:
```suggestion
    const { postId } = req.params;
```
http method중 delete를 통해 해당 API를 작성해주셨는데 delete 메소드는 원칙적으로 바디가 존재하지 않습니다. 예전에는 Body로 데이터를 보낼 수 없었지만, 근래에 들어 웹프레임워크 별로 혹은 버전별로 업데이트 되면서 body로 보내지는 데이터를 처리 할 수 있도록 일부 지원을 하기도 하지만, 기존에는 이런 부분이 지원되지 않았으므로 가급적 body가 아닌 params 혹은 query(금주 REST 세션에서 다룰 예정)로 구현하도록 수정해주세요.